src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -36,7 +36,7 @@ public class RachisAdminHandler : RequestHandler
                 command.VerifyCanExecuteCommand(ServerStore, context, isClusterAdmin);
 
 
-                var (etag, result) = await ServerStore.PutCommandAsync(command);
+                var (etag, result) = await ServerStore.SendToLeaderAsync(command);
```

Comment:
Nope, that is wrong. This is us getting a request from a node that think that _we_ are the leader.
If we aren't, we should throw.